post/views.py

```
@@ -0,0 +1,113 @@
+            
+class PostDetailView(APIView):
+
+    @swagger_auto_schema(
+            operation_id='게시글 상세 조회',
+            operation_description='게시글 1개의 상세 정보를 조회합니다.',
+            responses={200: PostSerializer}
+        )
+    def get(self, request, post_id): # get은 request 안쓴다. 그래서 회색빛.
+        try:
+            post = Post.objects.get(id=post_id) # 앞에와는 다르게 딱 하나만을 id를 이용해서 가져온다.
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)